astropy/modeling/tests/test_core.py

```diff
@@ -371,7 +371,7 @@ def test_custom_inverse_reset():
 
     class TestModel(Model):
         n_inputs = 0
-        outputs = ("y",)
+        _outputs = ("y",)
```

Comment:
Why change this?

Reply:
The comment I'm trying to honor from the `n_outputs` property definition indicates that `output` shouldn't be supported any more as a class attribute. Only a couple test cases were still relying on it, so I updated them for consistency. In effect however, they should still work after my cleanup, given that class attributes can be used as defaults for missing instance attributes.

Reply:
`outputs`, as well as `inputs`, are user facing attributes which can be changed on instances to reflect better the specific use case by assigning meaningful names. For example, if the input to a gaussian is wavelength, it will be much better for users to see this name instead of `x`.

Reply:
@nden sorry I might be confused by constant context switching but I can't seem to connect the dots here; are you completing my explanation or are you suggesting a change that I'm missing ?

Reply:
@neutrinoceros Sorry, I wasn't clear. I meant this needs to be in the public interface as end users can change it.